console/client/package.json

```diff
@@ -26,9 +26,9 @@
   ],
   "source": "index.html",
   "dependencies": {
-    "@bufbuild/connect": "0.12.0",
-    "@bufbuild/connect-web": "0.12.0",
-    "@bufbuild/protobuf": "1.3.0",
+    "@buf/connectrpc_eliza.connectrpc_es": "^1.1.2-20230913231627-233fca715f49.1",
```

Comment:
Isn't this a demo app?

https://connectrpc.com/demo/

Reply:
Lol, totally is. I got it from here:
https://connectrpc.com/docs/web/getting-started#generating-the-client